src/Kobo/System.cpp

```diff
@@ -359,12 +359,14 @@ KoboCanChangeBacklightBrightness()
 {
 #ifdef KOBO
   switch (DetectKoboModel()) {
-    case KoboModel::GLO_HD:
-    case KoboModel::LIBRA2:
-    case KoboModel::CLARA_2E:
-      return true;
-    default:
-      return false;
+  case KoboModel::GLO_HD:
+  case KoboModel::LIBRA2:
+  case KoboModel::CLARA_2E:
+  case KoboModel::CLARA_HD:
```

Comment:
This indent change has nothing to do with backlight, does it?

Reply:
It does have to do with the backlight but backlight brightness, not backlight colour.  I have done all the refactoring in a separate commit that preceeds the additions so it should be easy to follow.